src/main/kotlin/com/sourcegraph/cody/vscode/InlineAutocompleteItem.kt

```diff
@@ -1,8 +0,0 @@
-package com.sourcegraph.cody.vscode
-
-class InlineAutocompleteItem(
-    var insertText: String,
-    var filterText: String,
-    var range: Range,
-    var command: Command
```

Comment:
Can we remove `com.sourcegraph.cody.vscode.Command` btw?

Reply:
If its no longer being used anywhere, it can definitely be removed (now or later) i think 🙂